baybe/acquisition/acqfs.py

```diff
@@ -1,11 +1,106 @@
 """Available acquisition functions."""
 
+import math
 from typing import ClassVar
 
+import pandas as pd
+from attr.validators import optional as optional_v
 from attrs import define, field
-from attrs.validators import ge, instance_of
+from attrs.validators import ge, gt, instance_of, le
 
 from baybe.acquisition.base import AcquisitionFunction
+from baybe.searchspace import SearchSpace
+from baybe.utils.sampling_algorithms import (
+    SamplingMethod,
+    sample_numerical_df,
+)
+
+
+########################################################################################
+### Active Learning
+@define(frozen=True)
+class qNegIntegratedPosteriorVariance(AcquisitionFunction):
+    """Monte Carlo based negative integrated posterior variance.
+
+    This is particularly useful for active learning as it is a measure for global model
```

Comment:
Not sure if this necessary/correct to say, because it is in fact ONLY for active learning, since exploration-only... But if you think it's helpful, keep it

Reply:
so I just drop the 'particularly' ?